### ISSUE TITLE
Ability to read all the feeds I am subscriber of in the same time, a tab like "Following" but for feeds

### ISSUE THREAD

**Issue by bpavuk** [bluesky-social/social-app#3596]
**Is your feature request related to a problem? Please describe.**

Sometimes, I want to read multiple feeds simultaneously, but sometimes I don't know what do I want, but I know that tgis must be related to the topics I like.

**Describe the solution you'd like**

A separate tab called "All feeds" where the content from every feed I follow is gathered in a chronological order.

**Describe alternatives you've considered**

- I would like to build my own feed that works as I described, but therebis no Kotlin SDK for feeds right now
- The "For You" feed by SkyEye, but somehow it thinks I like furries, but I don't want to see them as often as every second post

**Comment by tfabris** [comment]
I have finally found this setting. Here is how to find it. 
- In the Bluesky phone app
- Upper left corner, press the icon with three lines (sometimes known as the hamburger menu)
- Select Settings
- Select Content and Media
- Select Following Feed Preferences 
- Select Experimental Show samples of your saved feeds in your following feed

**Comment by bpavuk** [comment]
Yeah, but I have zero control over frequency, and I'd like to dial it up.
